admin-dev/themes/new-theme/js/components/bloodhound/tokenizers.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import _ from 'lodash';
+import Bloodhound from 'typeahead.js';
+
+/**
+ * This comes from Bloodhound it allows to create tokenizer based on multiple fields from an object.
+ *
+ * @param tokenizer
+ * @returns {function(*=, ...[*]=): function(*): *[]}
+ */
+function getObjTokenizer(tokenizer) {
```

Comment:
@NeOMakinG I don't know if you saw this? It is used to allow searching into typeahead If you want to be able to search all the words and not the full (like men matches women and men) you can use this tokenizer
This has been introduced in categories component